src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
I was thinking to unite those commits so revert one would go away and from history stand point, it would be just support commit

> Now we're commenting on a Frankenstein diff...

nicely put, lol